test/test-functions

```diff
@@ -633,6 +633,7 @@ install_dmevent() {
     else
         inst_rules 10-dm.rules 13-dm-disk.rules 95-dm-notify.rules
     fi
+    [[ "$LOOKS_LIKE_SUSE" ]] && inst_rules 60-persistent-storage.rules 61-persistent-storage-compat.rules 99-systemd.rules
```

Comment:
Maybe,  `|| :` should be added at the end, that is,
```
[[ "$LOOKS_LIKE_SUSE" ]] && ... || :
```
or, use `if [[ ]]; then ...; fi`

Reply:
Thanks. I was wondering why the ci is failing...